src/modules/drawing/components/Color.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+    <div class="setting-container">
+        <span>{{ $t('modify_color_label') }}:</span>
+        <div class="color-select-box">
+            <div
+                v-for="c in colors"
+                :key="c.name"
+                :class="{ selected: color === c.fill }"
+                @click="() => onColorChange(c)"
+            >
+                <div :style="{ 'background-color': c.name }"></div>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+export default {
+    components: {},
+    props: {
+        colors: {
+            type: Array,
+            default() {
+                return []
+            },
+        },
+        color: {
```

Comment:
could be good to specify that this is the `currentColor`, it can be hard to know just by looking at the component why there is `colors` and `color`